src/lib.rs

```diff
@@ -2155,6 +2161,30 @@ fn ensure_libclang_is_loaded() {
 #[cfg(not(feature = "runtime"))]
 fn ensure_libclang_is_loaded() {}
 
+/// Error type for rust-bindgen.
+#[derive(Debug)]
+pub enum BindgenError {
```

Comment:
Consider `#[non_exhaustive]`? Or making a newtype around this, so that the fields are not publicly exposed?

Reply:
I think they should be exposed, and I added `#[non_exhaustive]` in `382841a`. Thanks!